arm-network/2016-09-01/swagger/network.json

```diff
@@ -4405,8 +4405,8 @@
           "description": "Health of backend server",
           "enum": [
             "Unknown",
-            "Up",
-            "Down",
+            "Healthy",
```

Comment:
This will be a breaking change to that API; is the SDK for this been released yet?